res/linuxPackaging.json

```diff
@@ -18,8 +18,8 @@
     "x-scheme-handler/ftp"
   ],
   "requires": [
-    "GConf2",
-    "libXScrnSaver",
+    "(GConf2 or gconf2)",
+    "(libXScrnSaver or libXss1)",
```

Comment:
@luixxiul it never worked on both Leap (stable) and Tumbleweed (rolling).
With this PR Tumbleweed is fixed and Leap remains unchabged (broken).

Maybe I should remove "fixes" tag?